build.savant

```diff
@@ -3,7 +3,7 @@
  */
 
 
-fusionauthWebsiteStyleVersion = "0.2.18"
+fusionauthWebsiteStyleVersion = "0.2.19-{integration}"
```

Comment:
You don't want to merge this to master, until 0.2.19 is released.

Reply:
Sorry, I realized that I should have referenced https://github.com/FusionAuth/fusionauth-style/pull/6 and said you need to pull that branch, do an `sb int` and then an `sb css` in `fusionauth-site` to see the change.